test/e2e/specs/domain-upsell/domain-upsell__my-home.ts

```diff
@@ -45,11 +45,13 @@ describe( DataHelper.createSuiteTitle( 'My Home: Domain upsell' ), function () {
 	} );
 
 	it( 'Get available domain', async function () {
-		selectedDomain = await myHomePage.suggestedDomainName();
+		const suggestedDomain = await myHomePage.suggestedDomainName();
+		expect( suggestedDomain ).toBeTruthy();
+		selectedDomain = suggestedDomain !== null ? suggestedDomain.toString() : '';
```

Comment:
I'm not 100% sure of this.
I did it because `validateCartItem` expect a string, but `textContent()` could be `null`.

What do you think about it, @worldomonation, @dpasque ?

Reply:
```suggestion
		selectedDomain = suggestedDomain ?? '';
```
Suggesting a shorter solution: When `suggestedDomain` is `null` it will be the empty string, otherwise the `suggestedDomain`.

Reply:
I think that looks just fine, and I really like Candy's shorthand version!

Although, you probably don't need the `expect` and the fallback. If `suggestedDomain` is `null`, the `expect` call with throw and exit early! 👍

Reply:
Thanks, @candy02058912, for the suggestion.
@dpasque, I thought the same, but without it, it fails at TeamCity because `cartCheckoutPage.validateCartItem` expect a string.

Reply:
Huh... Weird... 🤔  I think what you have is fine then! 👍